source/Mlos.Python/mlos/Optimizers/OptimizerBase.py

```diff
@@ -107,10 +113,30 @@ def optimum(self, optimum_definition: OptimumDefinition = OptimumDefinition.BEST
             if context is None:
                 raise ValueError(f"{optimum_definition} requires context to be not None.")
             return self._optimum_within_context(context=context)
+        if optimum_definition == OptimumDefinition.BEST_SERIES_ERROR_FUNCTION_MINIMIZATION:
+            if context is None:
+                raise ValueError(f"{optimum_definition} requires context to be not None.")
+            return self._optimum_with_series_difference(context=context)
         return self._prediction_based_optimum(parameters_df=parameters_df, optimum_definition=optimum_definition, alpha=alpha)
 
+    @trace()
+    def _optimum_with_series_difference(self, context: pd.DataFrame):
+        predicted_value_utility_function = SeriesPredictedValueUtilityFunction(
+            surrogate_model=self.surrogate_model,
+            objective=self.optimization_problem.objectives[0],
+            logger=self.logger
+        )
+        utility_optimizer = UtilityFunctionOptimizerFactory.create_utility_function_optimizer(
+            utility_function=predicted_value_utility_function,
+            optimizer_type_name=RandomSearchOptimizer.__name__,

```

Comment:
Why RandomSearchOptimizer?

Reply:
Also, at this point we definitely don't want the default config. We want some beefy config that would thoroughly explore the space for maxima.

Reply:
Thank you for pointing this out! That was a typo / bug